mdw-services/src/com/centurylink/mdw/service/data/process/EngineDataAccessDB.java

```diff
@@ -266,7 +266,19 @@ public Long createProcessInstance(ProcessInstance pi) throws SQLException {
         pi.setId(procInstId);
         return procInstId;
     }
-
+    public void setProcessElapsedTime(ProcessInstance pi) throws SQLException {
+        Long elapsedTime=getElapsedTime(pi.getId());
+        String query = "insert into INSTANCE_TIMING " +
+                      "(INSTANCE_ID, OWNER_TYPE, ELAPSED_MS) "+
+                       "values (?, ?, ?)";
+        Object[] args = new Object[3];
+        args[0] = pi.getId();
+        args[1] = pi.getOwner();

```

Comment:
OWNER_TYPE should be OwnerType.PROCESS_INSTANCE rather than the owner of the instance itself.